make-base/Makefile

```diff
@@ -0,0 +1,18 @@
+include ./stubs.mk
+
```

Comment:
unit tests for our make file functions, lol.  Yeah hopefully this doesn't get much bigger, but the find_projects_containing function is very helpful and warranted)

Reply:
Haha, cool! Do you plan to run these in CI, too?